recipes/tjpcov/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "tjpcov" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tjpcov-{{ version }}.tar.gz
+  sha256: fd4cd437105521e11c7f977d522cdfae75fed6d7e0c06afa599cc7eec8d7a4eb
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - python >=3.8                           # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Comment:
You don't need this for noarch packages.

```suggestion
```

Reply:
Fixed!